### PR TITLE
prototype 完整

### PR DESCRIPTION
**Pull request by ZJJ1991**
English version for prototype doc and add PoA doc skeleton

### CODE REVIEW

File: Prototypr(EN).md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+|setMaster|Set the new contract holder (Only the current contract holder has privilege to do this operation.)|self: The method caller' address<br>newMaster: New contract's holder's address.|
+|balance|Return the specified block's VET and VTHO balance.|self: The method caller' address.<br>blockNumber: Block number (The range of block number varies from 65535 ahead of latest block number to latest block number.)|
+|energy|Return the specified block's VTHO balance.|self: The method caller's address.<br>blockNumber: Block number (The range of block number varies from 65535 ahead of latest block number to latest block number.)|
+|hasCode|check the account type of the contract caller (contract account/normal account).|self: The method caller' address.|
+|storageFor|The account model's content stored in storage.|self: The method caller' address.<br>key: [Reference](https://solidity.readthedocs.io/en/latest/miscellaneous.html#layout-of-state-variables-in-storage)|
+|userPlan|Return the user planned balance and the recovery rate of plan.|self: The method caller' address.|
```
Comment (libotony) [inline Prototypr(EN).md]: Return the user planned balance and the recovery rate of plan. => Return the user plan, the maximum credit and recovery rate will be included in the user plan.

File: Prototypr(EN).md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+|setUserPlan|Set balance and recovery rate for all contract users.|self: The method caller' address.<br>credit: The available balance set to be used(Wei).<br>recoveryRate: The balance recovery rate based on per second(Wei).|
```
Comment (libotony) [inline Prototypr(EN).md]: Set balance and recovery rate => Set credit and recovery rate
The available balance set to be used => The maximum credit set to be used(Wei)

File: Prototypr(EN).md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+|isUser|Check whether the account is a contract user.|self: The method caller' address.<br>user: The specified user's address.|
+|userCredit|Return the specified user's available balance.|self: The method caller' address.<br>user: The specified user's address.|
```
Comment (libotony) [inline Prototypr(EN).md]: balance => credit

File: Prototypr(EN).md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+|addUser|Add the address to be contract user. Once successfully added, the default available balance and balance recovery rate will be set automatically. (Only the contract user and contract itself have privilege to call this function.)|self: The method caller' address.<br>user: The specified user's address.|
```
Comment (libotony) [inline Prototypr(EN).md]: (Only the contract user and contract itself have privilege to call this function.) => (Only the contract **master** and contract itself have privilege to call this function.)

File: Prototypr(EN).md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+|removeUser|Remove one contract user out of user list. (Only the contract user and contract itself have privilege to call this function.)|self: The method caller' address<br>user: The specified user's address.|
+|sponsor|Contract sponsor.|self: The method caller' address.<br>yesOrNo(bool): <br>true: Become the contract sponsor candidates.<br>false: Delete from contract sponsor candidates list.|
```
Comment (libotony) [inline Prototypr(EN).md]: Contract sponsor  =>Be a sponsor or not

File: Prototypr(EN).md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+## Contract Address
+ `0x000000000000000000000050726f746f74797065`
+
+>The above mentioned prototype contract address are same on both the test net and main net.
+
+##  Function
+
+|Function|Description|Parameter|
+|---|---|----|
+|master|By default, the contract deployer is the contract holder. The contract holder's address will be returned as a result of calling contract functions.|self: The method caller's address.|
```
[the remark this answers is not in the document]
Comment (ZJJ1991) [reply Prototypr(EN).md]: The latter is better in my opinion.

File: Prototypr(EN).md (hunk 4 above)
Comment (XJWX89) [inline Prototypr(EN).md]: the default available balance and balance recovery rate will be set automatically.=>the credits and recovery rate  will be set  as default automatically.

File: Prototypr(EN).md (hunk 2 above)
Comment (ZJJ1991) [reply Prototypr(EN).md]: The maximum credit set to be used(Wei) => The maximum balance that can be used(Wei)

File: Prototypr(EN).md [hunk 7]
```diff
@@ -6,25 +6,25 @@
...
-|isSponsor|Check whether the specified user is contract sponsor or not.|self: The method caller' address.<br>sponsor：The specified address that is going to be checked.|
-|selectSponsor|Select one candidate to be a contract sponsor from sponsor candidates list.|self: The method caller' address.<br>sponsor: The specified sponsor candidate's address in the contract sponsor candidates list.|
-|currentSponsor|Return the current contract sponsor.|self: The method caller' address.|
\ No newline at end of file
+|master|By default, the contract deployer is the contract master. The contract master's address will be returned as a result of calling this function.|self: The address to operate.|
+|setMaster|Set the new contract master (Only the current contract master has privilege to do this operation.)|self: The address to operate.<br>newMaster: New contract master's address.|
+|balance|Return the specified block's VTHO balance.|self: The address to operate.<br>blockNumber: Block number (The range of block number varies from 65535 ahead of latest block number to latest block number.)|
```
Comment (XJWX89) [inline Prototypr(EN).md]: VTHO balance => VET balance

File: Prototypr(EN).md [hunk 8]
```diff
@@ -6,25 +6,25 @@
...
+|isUser|Check whether the account is a contract user.|self: The address to operate.<br>user: The specified user's address.|
+|userCredit|Return the specified user's available credit.|self: The address to operate.<br>user: The specified user's address.|
+|addUser|Add the address to be contract user. Once successfully added, the default available balance and balance recovery rate will be set automatically. (Only the contract master and contract itself have privilege to call this function.)|self: The address to operate.<br>user: The specified user's address.|
+|removeUser|Remove one contract user out of user list. (Only the contract master and contract itself have privilege to call this function.)|self: The address to operate.<br>user: The specified user's address.|
+|sponsor|Be a sponsor or not.|self: The address to operate.<br>yesOrNo(bool): <br>true: Become the contract sponsor candidates.<br>false: Delete from contract sponsor candidates list.|
+|isSponsor|Check whether the specified user is a contract sponsor or not.|self: The address to operate.<br>sponsor：The specified address that is going to be checked.|
```
Comment (XJWX89) [inline Prototypr(EN).md]: specified user  -> specified address

**Pull request closed without merging**